pset-functions.js

```diff
@@ -136,6 +222,63 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+
+const wordPlayed = "javascript"
+
+scrabbleScore(wordPlayed)
+
+function scrabbleScore(word) {
+
+  let sum = 0
+
+  const Score1 = ["a","e","i","o","u","l","n","r","s","t"]
+  const Score2 = ["d","g"]
+  const Score3 = ["b","c","m","p"]
+  const Score4 = ["f","h","v","w","y"]
+  const Score5 = ["k"]
+  const Score8 = ["j","x"]
+  const Score10 = ["q","z"]
+
+  for (var position = 0; position < word.length; position++) {
+
+    let character = word.charAt(position)
+    //console.log(`character = ${word.charAt(position)}`)
+
+    if ( Score1.includes(character)) {
+      sum = sum + 1
+    }
+
+    else if  ( Score1.includes(character)) {
+      sum = sum + 1
+    }
+
+    else if ( Score2.includes(character)) {
+      sum = sum + 2
+    }
+
+    else if ( Score3.includes(character)) {
+      sum = sum + 3
+    }
+    else if ( Score4.includes(character)) {
+      sum = sum + 4
+    }
+    else if ( Score5.includes(character)) {
+      sum = sum + 5
+    }
+    else if ( Score8.includes(character)) {
+      sum = sum + 8
+    }
+    else if ( Score10.includes(character)) {
+      sum = sum + 10
+    }
+
+  }
+
+  console.log(`"${word}" has a Scrabble Score of ${sum}`)
+
+}
+
```

Comment:
@thetripler Good job here working through this problem. Good job using the function to check each character in the string and increment by the value of the letter. Good job using the array and the `includes` method to check each letter against the score array.
take a look at an alternate approach [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174)